src/builders/dev-server/index.ts

```diff
@@ -24,6 +26,17 @@ export class SingleSpaDevServer extends DevServerBuilder {
         SINGLE-SPA-ANGULAR: Angular dev server is serving application as a single module`);
         return webpackConfig;
     }    
+    _buildServerConfig(root, projectRoot, options, browserOptions) {
+        // @ts-ignore
+        const devServerConfig = super._buildServerConfig(root, projectRoot, options, browserOptions);
```

Comment:
typescript not liking me calling a private method

Reply:
I started down a similar route, but using a lot more duplication of the base classes which I didn't like. Can't say I'm a fan of fooling the compiler but sometimes you have to wonder why methods such as `_buildServerConfig` can't be made more accessible.

Reply:
Agreed. See related https://github.com/manfredsteyer/ngx-build-plus/pull/49 where this is what I ended up doing in ngx-build-plus to get things working.